src/packages/generate-license-file/test/utils/console.utils.spec.ts

```diff
@@ -50,6 +62,30 @@ describe("ConsoleUtils", () => {
       const firstCallThirdArg = mockLog.mock.calls[0][2];
       expect(firstCallThirdArg).toBe("param2");
     });
+
+    it("should stop and restart the logger if it is spinning", () => {
```

Comment:
I think test name is meant to be `restart the spinner` rather than logger. Same goes for below added tests!

Reply:
Yup, fixed